src/glue.c

```diff
@@ -1174,8 +1174,12 @@ void FuncDeclaration_toObjFile(FuncDeclaration *fd, bool multiobj)
          * 2. impact on function inlining
          * 3. what to do when writing out .di files, or other pretty printing
          */
-        if (global.params.trace)
+        if (global.params.trace &&
+            !(global.params.isOSX && fd->isCMain()))
```

Comment:
Do we ever want to trace C main? Probably not, so I guess the isOSX test may be eliminated.